tests/unit/config/test_configtypes.py

```diff
@@ -229,24 +229,30 @@ class GoodMappingSubclass(configtypes.MappingType):
 
     """A MappingType we use in TestMappingType which is valid/good."""
 
-    valid_values = configtypes.ValidValues('one', 'two')
-
     MAPPING = {
         'one': 1,
         'two': 2,
     }
 
+    def __init__(self, none_ok=False):
+        super().__init__(
+            none_ok,
+            valid_values = configtypes.ValidValues('one', 'two'))
+
 
 class BadMappingSubclass(configtypes.MappingType):
 
     """A MappingType which is missing a value in MAPPING."""
 
-    valid_values = configtypes.ValidValues('one', 'two')
-
     MAPPING = {
         'one': 1,
     }
 
+    def __init__(self, none_ok=False):
+        super().__init__(
+            none_ok,
+            valid_values = configtypes.ValidValues('one', 'two'))
```

Comment:
Why not just set `self.valid_values` separately like you do with `FlagListSubclass`?